From 88cf81bd9b04b26f8c0458082bb8450d859e6977 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:13:25 -0400
Subject: [alsa] Remove superfluous header inclusions in memalloc.c

Message-id: <140972823573-28-patchset-jkysela@redhat.com>
Patchwork-id: 90728
O-Subject: [RHEL 7.1 PATCH 027/442] ALSA: Remove superfluous header inclusions in memalloc.c
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

After cutting off the proc and page reservation codes, we don't need
many headers any longer.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: f5d8e6df4bfa749b4d029ba43ac8a027b1003c2a)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/core/memalloc.c b/sound/core/memalloc.c
index a57a627..5d7c0d0 100644
--- a/sound/core/memalloc.c
+++ b/sound/core/memalloc.c
@@ -22,17 +22,13 @@
  */
 
 #include <linux/module.h>
-#include <linux/proc_fs.h>
 #include <linux/init.h>
-#include <linux/pci.h>
 #include <linux/slab.h>
 #include <linux/mm.h>
-#include <linux/seq_file.h>
 #include <asm/uaccess.h>
 #include <linux/dma-mapping.h>
 #include <linux/genalloc.h>
 #include <linux/moduleparam.h>
-#include <linux/mutex.h>
 #include <sound/memalloc.h>
 
 
-- 
1.7.1